samples/client/petstore/java/feign10x/src/main/java/org/openapitools/client/auth/OAuth.java

```diff
@@ -81,19 +81,19 @@ public void apply(RequestTemplate template) {
         }
         // If first time, get the token
         if (expirationTimeMillis == null || System.currentTimeMillis() >= expirationTimeMillis) {
-            updateAccessToken();
+            updateAccessToken(template);
```

Comment:
@gaetPax I think you will need to update the mustache template as well: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/Java/libraries/feign/auth/OAuth.mustache#L86